app/Controller/UsersController.php

```diff
@@ -588,17 +590,7 @@ public function admin_view($id = null)
             unset($user['User']['authkey']);
         }
         if ($this->_isRest()) {
-            $user['User']['password'] = '*****';
```

Comment:
@iglocska Can you check if I break things here?  Looks like filtering in `admin_view()` and `view()` was different. This change unifies the filtering.